File: ktw_pad/Framework/Service/MobileRequest.cs
using System;
using System.Linq;
using System.Collections.Generic;
using System.Text;
using Framework;
using Json;

namespace Framework
{
    /// <summary>
    /// 请求对象
    /// </summary>
    [Serializable]
    public class MobileRequest
    {
        /// <summary>
        /// 画面id
        /// </summary>
        private string pageId;

        public string PageId
        {
            get { return pageId; }
            set { pageId = value; }
        }
        /// <summary>
        /// 出库代码
        /// </summary>
        private long whId;

        public long WhId
        {
            get { return whId; }
            set { whId = value; }
        }
        /// <summary>
        /// 用户ID
        /// </summary>
        private long userId;

        public long UserId
        {
            get { return userId; }
            set { userId = value; }
        }

        /// <summary>
        /// 货主ID
        /// </summary>
        private long ownerId;

        public long OwnerId
        {
            get { return ownerId; }
            set { ownerId = value; }
        }

        //参数

        private Dictionary<string, object> parameters = new Dictionary<string, object>();

        public Dictionary<string, object> Parameters
        {
            get { return parameters; }
            set { parameters = value; }
        }

    }
}

File: KTW_WMS/src/main/java/com/core/scpwms/server/mobile/bean/MobileRequest.java
package com.core.scpwms.server.mobile.bean;

import java.util.HashMap;
import java.util.Map;

import com.google.gwt.user.client.rpc.IsSerializable;

public class MobileRequest implements IsSerializable{
	private String pageId;
	
	private Long whId;
	
	private Long ownerId;
	
	private Long userId;
	
	//参数
	private Map<String,Object> parameters = new HashMap<String, Object>();

	public Map<String, Object> getParameters() {
		return this.parameters;
	}

	public void setParameters(Map<String, Object> parameters) {
		this.parameters = parameters;
	}

	public String getPageId() {
		return this.pageId;
	}

	public void setPageId(String pageId) {
		this.pageId = pageId;
	}

	public Long getWhId() {
		return this.whId;
	}

	public void setWhId(Long whId) {
		this.whId = whId;
	}

	public Long getUserId() {
		return this.userId;
	}

	public void setUserId(Long userId) {
		this.userId = userId;
	}

	public Long getOwnerId() {
		return ownerId;
	}

	public void setOwnerId(Long ownerId) {
		this.ownerId = ownerId;
	}

}
